components/RuiExpandCollapse/src/RuiExpandCollapse.ts

```diff
@@ -42,9 +36,16 @@ export class RuiExpandCollapse extends LitElement {
 	/* #region Properties*/
 
 	/**
-	 * Open property deals with the internal open/close
-	 * state. Mirrors the open attribute on the root element
+	 * The aria-level of the accordion item, defaults to to 2
 	 */
+	@property({type : Number})
+	public level?: number = 2;
```

Comment:
If you are defaulting this to `2` we can remove the `?` since it we can assume it will always have a value now

Reply:
leftover after changing approach. Will remove this